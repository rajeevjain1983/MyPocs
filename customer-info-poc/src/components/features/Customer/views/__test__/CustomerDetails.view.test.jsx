import React from "react";
import { shallow, configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import CustomerDetail from "../CustomerDetail.view";

configure({ adapter: new Adapter() });

describe("Customer Details View Rendered", () => {
  it("should renders correctly", () => {
    const props = {
      customer: {
        dateOfBirth: "08/08/1983",
        firstName: "Rajeev",
        lastName: "Jain",
        nameChanged: "yes",
        title: "Mr",
      },
      showModal: () => {},
    };

    const component = shallow(<CustomerDetail {...props} />);
    expect(component).toMatchSnapshot();
  });
});
