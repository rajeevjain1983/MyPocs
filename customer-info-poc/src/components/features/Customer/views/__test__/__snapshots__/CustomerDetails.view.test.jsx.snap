// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Customer Details View Rendered should renders correctly 1`] = `
<div>
  <h3>
    Below Customer detail saved successfully!
  </h3>
  <p>
    Name :
    Mr Rajeev Jain
  </p>
  <p>
    Date of Birth: 
    08/08/1983
  </p>
  <p>
    Has your name changed in the last 12 months? : 
    yes
  </p>
  <styled.button
    onClick={[Function]}
  >
    Ok
  </styled.button>
</div>
`;
